home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -8,7 +8,7 @@ info:
     url: https://opensource.org/licenses/mit-license.php
   contact:
     url: https://github.com/ita-social-projects/Home
-  version: 1.0.0
+  version: 1.0.1
```

Comment:
I'm not sure that we've started bumping this version. Let's postpone it until we decide on our versioning. Please revert.

Reply:
> I'm not sure that we've started bumping this version. Let's postpone it until we decide on our versioning. Please revert.

ok, I will revert